packages/components/src/mobile/bottom-sheet/test/cell.native.js

```diff
@@ -22,14 +24,15 @@ describe( '<BottomSheetCell />', () => {
 
 		const screen = render(
 			<BottomSheetCell
+				accessibilityLabel={ 'Text Input' }
```

Comment:
@fluiddot Ah, yep! I did not even consider that it would be just as easy to pass the a11yLabel as a prop here for testing. (Previously I was using testID.) Thanks for this suggestion!